src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx

```diff
@@ -187,6 +187,11 @@ const ProviderPanelLoader = ({
 		return <NoData onClose={onClose} provider={provider} />;
 	}
 	const { parse, name } = providers[provider];
+	/* eslint-disable-next-line @typescript-eslint/no-unsafe-argument --
+	 *
+	 * The `data` property on a `PlatformProviderResponse` is a true `any`,
```

Comment:
What's genuinely annoying about this situation is that our spec *does* specify that it is an object (which is all that `parse` wants).  The codegen, however, incorrectly translates `object` to `any`:

https://github.com/swagger-api/swagger-codegen/blob/be7735b8c32130a9ca317a955fde3116c9df4507/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractTypeScriptClientCodegen.java#L98

I'll open an issue because why not.

Reply:
Ah, interesting. In any case, of course I don't want to wait for that upstream to get resolved before this is merged in.

Reply:
Should I reword this comment, then, so it's more fitting / includes a link to the issue you're about to make?

Reply:
Wait, this commit from 5 years ago indicates they fixed it.

I wonder how old our codegen references are -- https://github.com/OpenAPITools/openapi-generator/commit/9368bea130c1bc5a0d039e228954c3e66ce89cc1

Reply:
@reefdog I think rewording would be reasonable; tbh the issue that I closed as wontfix probably should be reopened and you should link to that